editor/property_editor.cpp

```diff
@@ -701,17 +707,19 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
 					menu->add_separator();
 			}
 
-			menu->add_icon_item(get_icon("Load", "EditorIcons"), "Load", OBJ_MENU_LOAD);
+			menu->add_icon_item(get_icon("Load", "EditorIcons"), TTR("Load"), OBJ_MENU_LOAD);
 
 			if (!RES(v).is_null()) {
 
-				menu->add_icon_item(get_icon("EditResource", "EditorIcons"), "Edit", OBJ_MENU_EDIT);
-				menu->add_icon_item(get_icon("Del", "EditorIcons"), "Clear", OBJ_MENU_CLEAR);
-				menu->add_icon_item(get_icon("Duplicate", "EditorIcons"), "Make Unique", OBJ_MENU_MAKE_UNIQUE);
+				menu->add_icon_item(get_icon("EditResource", "EditorIcons"), TTR("Edit"), OBJ_MENU_EDIT);
+				menu->add_icon_item(get_icon("Del", "EditorIcons"), TTR("Clear"), OBJ_MENU_CLEAR);
+				menu->add_icon_item(get_icon("Duplicate", "EditorIcons"), TTR("Make Unique"), OBJ_MENU_MAKE_UNIQUE);
+				menu->add_separator();
+				menu->add_item(TTR("Show In File Manager"), OBJ_MENU_SHOW_IN_EXPLORER);
```

Comment:
It's `Show in File System` in 3.0. If it's exactly the same, translation will be shared.

![image](https://user-images.githubusercontent.com/26753932/30508639-4b0cbcba-9ac6-11e7-85ae-9e3ecd07ca67.png)

Reply:
3.0's "Show in File System" is show the resource file in filesystem dock.
My commit's "Show In File Manager" is show the resource file in OS's file manager,like windows explorer.
These are two different feture.
So I don't think they should share translation.

Reply:
May I ask why implement it differently from 3.0?

Reply:
I added this feature because it wasn't very convenient to develop the game.
If someone can move 3.0's "Show in File System" to 2.1, I think it will be a good helpful.

Reply:
I mean in 3.0 it highlights an item in FileSystem dock, but your code opens file manager. In my opinion, it might be better if they do the same.

Reply:
But these open two different  things.
I think in some languages, these two may have different translation.
In chinese,"Show in File System" should be translate to "在工程中显示".
"Show In File Manager" should be translate to "在资源管理器中显示".
So these are different.

Reply:
I realize that they do different things and have different translations. The thing is how it works is different from 3.0.

Reply:
This feature is not available in 2.1.
Anyone can remove it, if transplanted 3.0's "Show in File System".
I haven't studied the code of 3 in depth.
So I can't implement 3.0's "Show in File System",
Sorry for my poor english...